src/logic/operate.js

```diff
@@ -28,8 +31,12 @@ export default function operate(previous, next, operation) {
     case '=':
       value = prev;
       break;
+    case '÷':
+      if (currentNum === 0) this.error = "Can't divide by zero";
+      else calculation = previousNum / currentNum;
```

Comment:
const used in here seem undefined 🤔 

Hint: [Microverse operate.js template](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)

![image](https://user-images.githubusercontent.com/31897434/131220542-8889fab1-0964-4abd-a679-a195cdb612d4.png)


With the above example, kindly fix the `eslint` and add division by zero error message.